plugins/excel-adapter/src/main/java/org/polypheny/db/adapter/excel/ExcelFieldType.java

```diff
@@ -22,14 +22,15 @@
 import org.apache.poi.ss.usermodel.Cell;
 import org.polypheny.db.adapter.java.JavaTypeFactory;
 import org.polypheny.db.algebra.type.AlgDataType;
+import org.polypheny.db.catalog.exceptions.GenericRuntimeException;
 import org.polypheny.db.type.PolyType;
 
 /**
  * Type of a field in a Excel file.
  * <p>
  * Usually, and unless specified explicitly in the header row, a field is of type {@link #STRING}. But specifying the field type in the header row makes it easier to write SQL.
```

Comment:
This is copy-paste from the CSV (not by you but the person that created this adapter) and wrong here...

Reply:
I adjusted also this file, which is now quite similar to the csv one. While I am not happy with it I think we should move that change.